client/src/components/landing/HeroSection.tsx

```diff
@@ -19,6 +23,29 @@ const HeroSection: React.FC = () => {
           </Badge>
         </Link>
       </div>
+      <Button onClick={() => setShowLogin(true)}>Login</Button>
+      <Dialog open={showLogin} onClose={(val) => setShowLogin(val)} static={true}>
+        <DialogPanel>
+          <h2 className="text-lg font-semibold text-tremor-content-strong dark:text-dark-tremor-content-strong">Login</h2>
+          <div className="mx-auto max-w-sm space-y-8 mt-2">
+            <div>
+              <TextInput placeholder="Type your username here" error={error} errorMessage="Wrong username" />
```

Comment:
If I leave the username empty, there is no error message, do you want to control the validations on another phase? if yes, that is ok!

Reply:
we'll do that in the front-end logic UI